config/default.json

```diff
@@ -73,6 +73,7 @@
         "ExperimentalChannelOrganization": false,
         "EnableAPITeamDeletion": false,
         "ExperimentalEnableHardenedMode": false,
+        "DisableLegacyMFA": true,
```

Comment:
Have we confirmed with PMs that it's okay to disable legacy MFA by default right now? This will break any old mobile apps

Reply:
The idea was that since we only set it default to false in the`default.json`, but in the code set it to `false` if it's not installed, it will only be disabled by default for new installations which configuration is based on the `default.json`. If you are migrating to a new version with your previous config, it will default to `false` so there shouldn't be any issues.

Reply:
Paging @wiersgallak for review.

Reply:
Oh, I see. That makes sense then. I didn't notice that you made it so that it would default to false on upgrade or true on a new install.

Reply:
Daniel and I discussed this change makes sense to move customers to the recommended end-point, but still provide backwards compatibility if needed.